MekHQ/src/mekhq/gui/InterstellarMapPanel.java

```diff
@@ -329,33 +329,36 @@ public void actionPerformed(ActionEvent ae) {
                     popup.add(item);
                     JMenu menuGM = new JMenu("GM Mode");
                     item = new JMenuItem("Move to selected planet");
-                    item.setEnabled(selectedPlanet != null && campaign.isGM());
-                    if (selectedPlanet != null) {// only add if there is a planet to center on
+                    item.setEnabled(selectedSystem != null && campaign.isGM());
+                    if (selectedSystem != null) {// only add if there is a planet to center on
                         item.addActionListener(new ActionListener() {
                             @Override
                             public void actionPerformed(ActionEvent ae) {
-                                campaign.getLocation().setCurrentPlanet(selectedPlanet);
+                                campaign.getLocation().setCurrentSystem(selectedSystem);
                                 campaign.getLocation().setTransitTime(0.0);
                                 campaign.getLocation().setJumpPath(null);
                                 jumpPath = new JumpPath();
-                                center(selectedPlanet);
+                                center(selectedSystem);
                                 hqview.refreshLocation();
                             }
                         });
                     }
                     menuGM.add(item);
+                    /*
+                     * TODO: re-enable this later
```

Comment:
Hanging TODO

Reply:
I was planning on leaving the ability to edit planetary system information to a separate PR to avoid having this one get bogged down even further.